pep-0676.rst

```diff
@@ -121,69 +128,84 @@ The following redirect rules must be created from the `python.org`_ domain:
         return 308 https://peps.python.org/;
     }
 
-Redirects must be implemented to preserve `URL fragments`_ for backward
-compatability purposes.
+Redirects MUST be implemented to preserve `URL fragments`_ for backward
+compatibility purposes.
+
 
 Backwards Compatibility
 =======================
 
-Due to server-side redirects to new canonical URLs, there are no backwards
-compatability concerns. Links in previously published materials referring to
-any old URL scheme will be guaranteed to work.
+Due to server-side redirects to new canonical URLs, links in previously
+published materials referring to the old URL scheme will be guaranteed to work.
+Furthermore, aside from the resolved bugs and new implemented features, all
+existing PEPs will still render as they did before with the new backend.
+Therefore, this poses no major backwards compatibility concerns.
```

Comment:
> will still render as they did before with the new backend

You can argue this both ways (the outputted HTML is different, but the words are the same). I'm not sure it adds anything, I guess.

```suggestion
Due to server-side redirects to new canonical URLs, links in previously
published materials referring to old URL schemes will be guaranteed to work.
We have not identified any other backwards compatibility concerns.
```

Reply:
While I understand my original wording wasn't the best, I have to disagree with your dismissal of its importance overall. As a reader and reviewer of the PEP, my biggest backward compat concern aside from the URL change would be whether it causes any major rendering regressions in displaying previously valid syntax/formatting used in existing PEPs. Therefore, I certainly wouldn't dismiss this entirely as a backward compat concern. While those of us involved are well aware it doesn't, making this explicit helps ensure this is clear to everyone. What about (also fixed a missing definite article):

```suggestion
Due to server-side redirects to the new canonical URLs, links in previously
published materials referring to the old URL scheme will be guaranteed to work.
Furthermore, while the theme and presentation may differ, all existing PEPs
will still render correctly as they did before with the new backend.
Therefore, this poses no major backwards compatibility concerns.
```